internal/handlers/handlers.go

```diff
@@ -0,0 +1,42 @@
+package handlers
+
+import (
+	"errors"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	linker "github.com/kisanetik/learn_go_inc1/internal/app"
+)
+
+func MethodPost(res http.ResponseWriter, req *http.Request) {
+	body, _ := io.ReadAll(req.Body)
+	if len(body) == 0 {
+		res.WriteHeader(http.StatusBadRequest)
+	}
+	location := linker.CompressURL(string(body))
+	res.WriteHeader(http.StatusCreated)
+	res.Write([]byte(location))
+}
+
+func MethodGet(res http.ResponseWriter, req *http.Request) {
+	tFile, err := os.CreateTemp("", "")
+	if err != nil {
+		panic(err)
+	}
+	filename := filepath.Dir(tFile.Name()) + req.RequestURI
+	if _, err := os.Stat(filename); err == nil {
+		data, fErr := os.ReadFile(filename)
+		if fErr != nil {
+			panic(fErr)
```

Comment:
It would be better if you use the 500 response.

Reply:
ok